develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.RawType.md

```diff
@@ -58,46 +56,36 @@ The following characters and digits:
 - e or E or ' ' (space)
 - a-f or A-F (if Protocol.Params.Param.Interprete.Base is 16)
 
-In the following example, the value -3 will be saved in database as 2D 33:
-
-```xml
-<RawType>numeric text</RawType>
-```
+The value -3 will be saved as string in the database as "2D33"
 
 ### only others
 
 Values made up of different types.
 
 See Protocol.Params.Param.Interprete.[Others](xref:Protocol.Params.Param.Interprete.Others).
 
-### other
-
-ASCII characters, both numeric and alphanumeric.
+> [!NOTE]
+> only others is depricated. It can still be used but is not recommended for new developments. We advice to use other in combination with [exception](xref:Protocol.Params.Param.Interprete.Exceptions.Exception) instead.
 
-Example:
+### other
 
-```xml
-<RawType>other</RawType>
-```
+any characters, both alphanumeric and non-alphanumeric.
 
 ### signed number
 
 When a next param is defined in Interprete.Type, the result will be the same as with unsigned number. When a fixed length is defined in Interprete.Type, the first half of the values will be positive number, and second half will be negative number.
```

Comment:
I was really confused when I read "the first half" for the first time. Why not just specify the way it is, because 0 to UINT.Max doesn't map on INT.MIN to INT.MAX. With signed, if the most significant bit is high, the value will be negative according to the ones' complement: https://en.wikipedia.org/wiki/Signed_number_representations#Ones'_complement